starter-code/public/index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>Llamas are landing</title>
```

Comment:
🐪(they don't have a llama emoji yet)

Reply:
aha would love to be able to do a PR to create one, ultimate interview personal perk aha